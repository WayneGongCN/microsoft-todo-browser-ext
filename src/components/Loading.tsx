import { Container } from '@material-ui/core';
import React, { useCallback } from 'react';

const Loading: React.FC = () => {
  return (<Container>Loadong ...</Container>);
};

export default Loading;
