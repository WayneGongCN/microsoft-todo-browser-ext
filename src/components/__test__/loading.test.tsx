/* eslint-disable @typescript-eslint/ban-ts-comment */
import React from 'react';
import snapshotRender from 'react-test-renderer';
import Loading from '../Loading';


describe('Snapshot test', () => {
  test('normal render', () => {
    const tree = snapshotRender.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot('normal');
  });
});
