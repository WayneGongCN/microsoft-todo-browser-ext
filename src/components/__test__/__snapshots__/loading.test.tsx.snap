// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test normal render: normal 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
>
  Loadong ...
</div>
`;
